### PR TITLE
Fixes #8716: Remove 'isxxx' and add priority in Directive from api doc

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://www.rudder-project.org/redmine/issues/8716

### CODE REVIEW

File: src/api_v2/directive.md [hunk 1]
```diff
@@ -246,6 +246,7 @@ https://github.com/Normation/rudder/blob/master/rudder-web/src/main/scala/com/no
     @apiParam (Mono) {UUID}              [id=autoGenerated]                          Define the id of the Directive to create, Rudder will generate an id if this is missing. You should only use it for specific use case.
     @apiParam (Mono) {Boolean}           [enabled=true]                              Enable the Directive at creation. This parameter can be overriden.
     @apiPAram (Mono) {TechniqueVersion}  [techniqueVersion=latest Technique version] In which version your tehcnique should be created.
```
Comment (jooooooon) [inline src/api_v2/directive.md]: Looks like a typo here: apiP**A**ram (should be apiP**a**ram)

File: src/api_v2/directive.md [hunk 2]
```diff
@@ -246,6 +246,7 @@ https://github.com/Normation/rudder/blob/master/rudder-web/src/main/scala/com/no
...
+    @apiParam (Mono) {Int}               [priority]                                  Directive priority. Determine which value to use if there is Directives based on the same technique applied to a Node.
```
Comment (jooooooon) [inline src/api_v2/directive.md]: s/Determine/Determines/

Comment (jooooooon) [inline src/api_v2/directive.md]: s/there is/there are/

Comment (jooooooon) [inline src/api_v2/directive.md]: s/Directives/several Directives/

**Comment by jooooooon** [comment]
Looks good, but shouldn't we mention somewhere that these *used* to be called isEnabled, isSystem and isAcceptable? It seems weird just rewriting history. We could also add a quick paragraph explaining that as of Rudder X.Y.Z there is an option to use backwards compatibles names that is disabled by default on new installs.

**Comment by jooooooon** [comment]
Ping @VinceMacBuche ?

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by jooooooon** [comment]
:+1:

**Pull request merged**